google/cloud/storage/internal/grpc/default_options_test.cc

```diff
@@ -132,11 +132,17 @@ TEST(DefaultOptionsGrpc, MetricsPeriod) {
 }
 
 TEST(DefaultOptionsGrpc, MinMetricsPeriod) {
-  auto const options =
+  auto const o0 =
       DefaultOptionsGrpc(Options{}.set<storage_experimental::GrpcMetricsPeriod>(
-          std::chrono::seconds(1)));
-  EXPECT_GE(options.get<storage_experimental::GrpcMetricsPeriod>(),
-            std::chrono::seconds(60));
+          std::chrono::seconds(0)));
+  EXPECT_GT(o0.get<storage_experimental::GrpcMetricsPeriod>(),
+            std::chrono::seconds(0));
```

Comment:
I think this should check for >= 5 seconds. Any less, and some of the exports will fail.

Unless you want to argue that "The GCM minimum may change, so we cannot say what it is, but we know it is greater than 0".

Reply:
Yes, I would like to argue that we don't care that much:

https://testing.googleblog.com/2015/01/testing-on-toilet-change-detector-tests.html